dashboard/src/components/ErrorAlert/ErrorAlert.tsx

```diff
@@ -8,7 +8,8 @@ import { CustomError } from "shared/types";
 import "./ErrorAlert.css";
 
 export interface IErrorAlert {
-  children: CustomError | Error | string;
+  error: CustomError | Error;
```

Comment:
When changing to an `error` prop, given that this is an `ErrorAlert` component (ie. something that should be used when you have an error), it no longer seemed to make sense to support a plain string here since (a) why use an `ErrorAlert` in that case, and (b) even if you wanted to, you could put the string as the child.

Reply:
There could be use cases where we need to show a plain text message as an error on UI (red, etc.), but agree with you that it can be done with `(b)`.